build.sh

```diff
@@ -131,8 +131,12 @@ chmod 755 $DESTINATION/sites/default
 
 # Inculde copies of the settings files that were used to build the site, for reference
 SETTINGS_SITE="$DESTINATION/profiles/$PROJECT/settings"
+
+# replace PROJECT with actual project name in local.settings.php
+sed -i '' "s/PROJECT/$PROJECT/g" $DESTINATION/profiles/$PROJECT/scripts/settings/local.settings.php
```

Comment:
@mikemccaffrey please review this and decide what you think is best here. This is an older line of code that I think we either may not need now, or may want to add an if statement for.